src/pages/mypage/components/profileSection/ProfileSection.tsx

```
@@ -0,0 +1,39 @@
+  return (
+    <section className={styles.container}>
+      <div className={styles.profileBox}>
+        <div className={styles.iconWrapper}>
+          <ProfileIcon />
+        </div>
+
+        <div
+          className={styles.creditWrapper}
+          onClick={onChargeClick}
+          role="button"
+        >
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)